src/Components/test/E2ETest/Infrastructure/ServerFixtures/TrimmingServerFixture.cs

```diff
@@ -0,0 +1,57 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Reflection;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Testing;
+
+namespace Microsoft.AspNetCore.Components.E2ETest.Infrastructure.ServerFixtures;
+
+// REVIEW: Should this be merged into BasicTestAppServerSiteFixture? Is there any case where we wouldn't
+// want to trim BasicTestAppServerSiteFixture tests when TestTrimmedOrMultithreadingApps is true?
```

Comment:
@dotnet/aspnet-blazor-eng I'm curious what everyone thinks about this.

Reply:
I would say that if other tests don't break, we could enable it, but I suspect we wanted to have this isolated. I believe in the past we would test all the apps with trimming during release. I don't know if/when that changed, but I'm ok if we bring that back.